package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "loose-rgb",
-  "version": "1.0.6",
+  "type": "module",
```

Comment:
I heard not to use `"type": "module"`, but... I am not adding a plugin just to rename files to `.mjs` at this point